### PR TITLE
Update Cumulus workflow usage doc to reference Cumulus documentation

### PR DESCRIPTION
**Pull request by Jkovarik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,27 +92,11 @@ The compiled task code, the message parser uber-jar, the cumulus message adapter
...
-  source: '../cumulus-message-adapter-java/deploy/'
-  useMessageAdapter: true
-  runtime: java8
-  memory: 256
-```
-
-The source points to a folder with the compiled .class files and dependency libraries in the Lambda Java zip folder structure (details [here](https://docs.aws.amazon.com/lambda/latest/dg/create-deployment-pkg-zip-java.html)), not an uber-jar.
-
-The deploy folder referenced here would contain a folder 'test_task/task/' which contains Task.class and TaskLogic.class as well as a lib folder containing dependency jars. The Cumulus Message Adapter zip would be added at the top level by the deployment step and that folder zipped and deployed to Lambda. 
+For documenation on how to utilize this package in a Cumulus Deployment, view the [Cumulus Workflow Documenation](https://nasa.github.io/cumulus/docs/workflows/input_output).
```
Comment (laurenfrederick) [inline README.md]: I think there is some Java-specific documentation in there that we don't want to lose. We could move it to the main documentation. Although with the CMA as a layer, could we just point the task to the uberjar? 

We should at least point them more specifically to here: https://nasa.github.io/cumulus/docs/workflows/lambda#deploy-a-lambda

and maybe update that to have a Java example.

**Review by laurenfrederick — CHANGES_REQUESTED** [review]
Commented on the doc update.

File: README.md (hunk 1 above)
Comment (Jkovarik) [reply README.md]: > Although with the CMA as a layer, could we just point the task to the uberjar?

Not really - the CMA as layer doesn't include the client lib (this package), the CMA as layer means the underlying (nasa/cumulus-message-adapter) python utility is present on the system in /opt.       Including the client libs is needed to make use of that.

Comment (Jkovarik) [reply README.md]: @laurenfrederick 

> We should at least point them more specifically to here: https://nasa.github.io/cumulus/docs/workflows/lambda#deploy-a-lambda

How about both?   The input/output documentation seems like it contains more input/output expectations and the like *with* configuration option examples 🤔   

It's also complicated by the IX team rewriting all of this documentation a sprint from now as well. 

Anyway, how about: 

https://github.com/nasa/cumulus/pull/1206

and 

df787d9

?

**Review by laurenfrederick — APPROVED** [review]

**Pull request merged**